packages/ncids-css/scss/components/nci-header/internal/_nci-navbar.scss

```diff
@@ -17,17 +17,29 @@
 			width: 100%;
 
 			a {
-				color: color("ink");
+				color: color('ink');
 				display: block;
 				height: 38px;
 				width: auto;
 				text-decoration: none;
+				@include at-media($theme-header-min-width) {
+					height: 50px;
+				}
+			}
+
+			picture {
+				display: block;
 			}
 
 			img {
-				height: auto;
-				max-width: 100%;
+				height: 38px;
 				max-height: 38px;
+				max-width: 100%;
+
+				@include at-media($theme-header-min-width) {
+					height: 50px;
```

Comment:
@dlescarbeau Great question. I just asked Paul. However 38 and 50 are not any of the USWDS units. So in hindsight, yeah it would have been nice if these had been USWDS sizes.